src/pages/admin/AdminPanel.js

```diff
@@ -1,136 +1,133 @@
-import React, { Component, useState, useEffect } from 'react';
-import AdminHomepage from './AdminHomepage';
+import React, { useState, useEffect } from 'react';
+import { BiCopyright } from 'react-icons/bi';
+import SideNav, { NavItem, NavIcon, NavText } from '@trendmicro/react-sidenav';
 import styled from 'styled-components';
+import { Routes as Switch, Route, useNavigate, Outlet } from 'react-router-dom';
+import AdminHomepage from './AdminHomepage';
 import UserManager from './UserManager';
-import { barChartIcon, userIcon, newsIcon, formIcon, feedbackIcon, socialMediaIcon, forumIcon } from "../../assets/index.js";
-import { Routes as Switch, Route, useNavigate } from 'react-router-dom';
-import SideNav, { NavItem, NavIcon, NavText } from '@trendmicro/react-sidenav';
+import {
+  barChartIcon, userIcon, newsIcon, formIcon, feedbackIcon, socialMediaIcon, forumIcon
+} from '../../assets/index';
 import AdminNavbar from './AdminNavbar';
-import {BiCopyright} from 'react-icons/bi';
 import './AdminPanel.scss';
 
 const Main = styled.main`
         position: relative;
         //overflow: scroll;
         transition: all .15s;
-        margin-left: ${props => (props.expanded ? 240 : 64)}px;
+        margin-left: ${(props) => (props.expanded ? 240 : 64)}px;
     `;
 
 function AdminPanel(props) {
-    const navigate = useNavigate();
-    const [selected, setSelected] = useState('dashboard');
-    const [expanded, setExpanded] = useState(false);
+  const navigate = useNavigate();
+  const [selected, setSelected] = useState('dashboard');
+  const [expanded, setExpanded] = useState(false);
 
-    const renderMainPage = () => {
-        return (
-            <>
-                <Switch>
-                    <Route exact path="/admin/dashboard" component={AdminHomepage} />
-                    <Route path="/admin/manageUsers" component={UserManager} />
-                </Switch>
-            </>
-        )
-    };
-
-    useEffect(() => {
-        props.toggleNavbar(false);
-        props.toggleFooter(false);
-        onSelect(selected);
-        return () => {
-            props.toggleNavbar(true);
-            props.toggleFooter(true);
-        };
-    }, [props]);
-
-    const onSelect = (selected) => {
-        setSelected(selected);
-        const to = '/admin/' + selected;
-        navigate(to);
-    };
+  const onSelect = (selected) => {
+    //const { history } = props;
```

Comment:
Do not commit commented code, same below.

Reply:
Removed